ntoskrnl/config/i386/cmhardwr.c

```diff
@@ -376,6 +386,32 @@ CmpInitializeMachineDependentConfiguration(IN PLOADER_PARAMETER_BLOCK LoaderBloc
                         (Prcb->CpuStep >> 8),
                         Prcb->CpuStep & 0xff);
             }
+#elif defined(_M_AMD64)
```

Comment:
I was looking at Geoff Chappell's website to see the differences between these `CpuID`, `CpuVendor` etc. fields.
Turns out that `CpuID` field in x64 should always be TRUE: I'm wondering whether we could add somewhere some
`ASSERT(Prcb->CpuID == TRUE);` 🤔

Reply:
Don't have a checked build x64 kernel at hand, so don't know if assertion like that exists. Also in [this function](https://git.reactos.org/?p=reactos.git;a=blob;f=ntoskrnl/ke/amd64/cpu.c;hb=1f6cd380509e7eab8cb653a748ff947fbf8d6974#l43) the code permanently sets it to TRUE.

Reply:
ah ok then no need for any assertion.